tests/lightning_duplicate_detection/config1.toml

```diff
@@ -0,0 +1,30 @@
+[tikv-importer]
+backend = "local"
+duplicate-detection = true
+
+[checkpoint]
+enable = true
+schema = "tidb_lightning_checkpoint1"
+driver = "mysql"
+
+[[mydumper.files]]
+pattern = '(?i).*(-schema-trigger|-schema-post)\.sql$'
+type = 'ignore'
+
+[[mydumper.files]]
+pattern = '(?i)^(?:[^/]*/)*([^/.]+)-schema-create\.sql$'
+schema = '$1'
+type = 'schema-schema'
+
+[[mydumper.files]]
+pattern = '(?i)^(?:[^/]*/)*([^/.]+)\.(.*?)-schema\.sql$'
+schema = '$1'
+table = '$2'
+type = 'table-schema'
+
+[[mydumper.files]]
+pattern = '(?i)^(?:[^/]*/)*([^/.]+)\.(.*?)\.0\.sql$'
+schema = '$1'
+table = '$2'
+key = '0'
```

Comment:
(optional)

perhaps easier just use two distinct `data-source-dir` with `no-schema = true`.

Reply:
I do not understand...